sippy-ng/src/component_readiness/CompCapRow.js

```diff
@@ -85,5 +86,5 @@ CompCapRow.propTypes = {
   results: PropTypes.array.isRequired,
   columnNames: PropTypes.array.isRequired,
   filterVals: PropTypes.string.isRequired,
-  accessibilityMode: PropTypes.bool.isRequired,
+  accessibilityMode: PropTypes.bool,
```

Comment:
Debugger was throwing a warning that accessiblityMode was marked required but undefined.  Looks like [default](https://github.com/openshift/sippy/blob/e5957bd255c9570802fd21327f9e8e5f7edcfcbf/sippy-ng/src/component_readiness/CompReadyUtils.js#L123) is set so removed isRequired

Reply:
I didn't see the same warning when I tried running the debugger from the prod ui.  I was thinking the initial default value was undefined at the time the debugger was loading and it was throwing the warning.

Reply:
I backed these changes out since I don't see them in production.

When debugging locally I see:
```
Warning: Failed prop type: The prop `accessibilityMode` is marked as required in `CompSeverityIcon`, but its value is `undefined`.
```

But we can leave this for another day if it is a regular issue.

Reply:
Missed one